<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>State</class>
 <widget class="QWidget" name="State">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>State</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QCheckBox" name="checkBoxStateInterface">
       <property name="text">
        <string>state interface</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="3" column="1">
    <widget class="QPlainTextEdit" name="plainTextEditStateInterface"/>
   </item>
   <item row="1" column="0">
    <widget class="QCheckBox" name="checkBoxState">
     <property name="text">
      <string>state</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="lineEditState"/>
   </item>
   <item row="0" column="0">
    <widget class="QCheckBox" name="checkBoxContext">
     <property name="text">
      <string>context</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="lineEditContext"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
